pkg/runtime/runtime.go

```diff
@@ -582,10 +586,26 @@ func (a *DaprRuntime) appHealthReadyInit(ctx context.Context) (err error) {
 	return nil
 }
 
+func setActorRuntimeForBackend(ctx context.Context, be backend.Backend, actorRuntime actors.ActorRuntime) error {
+	if abe, ok := be.(*wfengine.ActorBackend); ok {
+		log.Info("Configuring workflow engine with actors backend")
+		return abe.SetActorRuntime(actorRuntime, ctx)
+	}
+
+	return nil
+}
+
 func (a *DaprRuntime) initWorkflowEngine(ctx context.Context) {
 	wfComponentFactory := wfengine.BuiltinWorkflowFactory(a.workflowEngine)
 
-	a.workflowEngine.SetActorRuntime(a.actor)
+	if a.workflowEngine.BackendType == wfengine.ActorBackendType {
+		err := setActorRuntimeForBackend(ctx, a.workflowEngine.Backend, a.actor)
+
+		if err != nil {
+			log.Errorf("could not set actor runtime for backend: %v", err)
```

Comment:
It looks like we're just swallowing the error but continuing to run. Is that going to be okay or do we need to stop the startup of the Dapr runtime?

Reply:
SetActorRuntime is simplified as it was earlier, where we only initializing the actor and nothing else. So, returning the error doesn't add any value as it always be nil. So, removing this error check.